webapp/channels/src/components/channel_bookmarks/channel_bookmarks_create_modal.tsx

```diff
@@ -37,6 +37,9 @@ import './bookmark_create_modal.scss';
 import CreateModalNameInput from './create_modal_name_input';
 import {useCanUploadFiles} from './utils';
 
+const MAX_LINK_LENGTH = 1024;
+const MAX_TITLE_LENGTH = 64;
```

Comment:
Should be enforced on mobile as well

Reply:
Created https://mattermost.atlassian.net/browse/MM-61110